### PR TITLE
Init Base NFT Folder

### PR DESCRIPTION
**Pull request by MSilb7**
Copy Optimism NFT -> Base.

Find and replace Optimism for Base. This is a prereq for https://github.com/duneanalytics/spellbook/pull/3896

This depends on: https://github.com/duneanalytics/spellbook/pull/3898 (Some necessary files copied over)

### CODE REVIEW

**Comment by MSilb7** [comment]
Do these tag errors mean I'm forced to migrate all upstream spells to dunesql? Same errors as https://github.com/duneanalytics/spellbook/pull/3898 & https://github.com/duneanalytics/spellbook/pull/3842

File: models/tokens/base/tokens_base_nft_curated_legacy.sql
Comment (jeff-dude) [inline models/tokens/base/tokens_base_nft_curated_legacy.sql:1]: can we use the dummy approach here? no need to build on spark, as it's a new spell

File: models/tokens/base/tokens_base_nft_legacy.sql
Comment (jeff-dude) [inline models/tokens/base/tokens_base_nft_legacy.sql:1]: can we use the dummy approach here? no need to build on spark, as it's a new spell

File: models/tokens/base/tokens_base_nft_standards_legacy.sql
Comment (jeff-dude) [inline models/tokens/base/tokens_base_nft_standards_legacy.sql:1]: can we use the dummy approach here? no need to build on spark, as it's a new spell

File: models/tokens/base/tokens_base_nft_curated.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{ config(alias = alias('nft_curated'), tags=['static', 'dunesql']) }}
+
+SELECT
+  contract_address, name, cast(NULL as varchar) as symbol
```
Comment (jeff-dude) [inline models/tokens/base/tokens_base_nft_curated.sql:4]: you'll never use symbol? or will edit later?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/tokens/base/tokens_base_nft_standards_legacy.sql
Comment (MSilb7) [reply models/tokens/base/tokens_base_nft_standards_legacy.sql:1]: editing

File: models/tokens/base/tokens_base_nft_curated_legacy.sql
Comment (MSilb7) [reply models/tokens/base/tokens_base_nft_curated_legacy.sql:1]: done

**Comment by MSilb7** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MSilb7** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MSilb7** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MSilb7** [comment]
recheck

**Comment by MSilb7** [comment]
recheck

File: models/tokens/base/tokens_base_nft_legacy.sql
Comment (jeff-dude) [inline models/tokens/base/tokens_base_nft_legacy.sql:1]: we can continue to simplify here. no need to mimic the logic / joins between two legacy files. rather, just continue to use `select 1` on all legacy files

if that means you can simply the others too (`tokens_base_nft_standards_legacy`, `tokens_base_nft_curated_legacy`) to write less columns, that is good too. i assume you had added multiple columns in those to help the joins here. that won't be needed once this one is just `select 1` again

Comment (jeff-dude) [reply models/tokens/base/tokens_base_nft_legacy.sql:1]: can also simplify config block to just schema/alias/tags. as it stands now, with table materialization, it'll run daily when we don't need it to. should also avoid exposing in data explorer.

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/tokens/base/tokens_base_nft_legacy.sql
Comment (MSilb7) [reply models/tokens/base/tokens_base_nft_legacy.sql:1]: updating

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**